src/background/http-instrument.ts

```diff
@@ -231,12 +232,22 @@ export class HttpInstrument {
     );
     */
 
+    const tab =
+      details.tabId > -1
+        ? await browser.tabs.get(details.tabId)
+        : { windowId: undefined, incognito: undefined, url: undefined };
+
     // http_requests table schema:
     // id [auto-filled], crawl_id, url, method, referrer,
     // headers, visit_id [auto-filled], time_stamp
     const update = {} as HttpRequest;
 
+    update.incognito = boolToInt(tab.incognito);
     update.crawl_id = crawlID;
```

Comment:
My original idea was also to combine the attributes, but I wouldn't want to remove or change the crawl_id attribute before having this discussion. Main question is whether or not crawl_id is shared between many browser sessions or if it is unique to one browser spawn? If the latter, we should indeed merge the two attributes.

Reply:
It is shared between sessions (it's essentially a "crawler index" id generated by the TaskManager to identify which one of the N browsers it is managing is being interacted with). However that consistency is not a property we rely on and we could instead generate a new crawl_id every time the browser is restarted.

Reply:
A name of `session_id` is a bit too generic and suggests a relation with an active session id on the currently visited site. It could be called something like `browser_process_uuid`, `browser_process_id`, `browser_launch_uuid`, `browser_launch_id`.

Reply:
Let's go with `browser_launch_id`. Filed https://github.com/mozilla/openwpm-webext-instrumentation/issues/29 for this.